pandunia-lekse.md

```diff
@@ -0,0 +1,46 @@
+The pandunia-lekse.csv in not valid by RFC 4180.
```

Comment:
This file should be inside the engli folder because it should be translated to other languages also.

Reply:
Should I move it in my repo? I'm not experienced in Git

Reply:
Done it in my repository.